runtime/moonbeam/tests/integration_test.rs

```diff
@@ -34,6 +38,58 @@ use precompiles::MoonbeamPrecompiles;
 use sp_core::{Public, H160, U256};
 use sp_runtime::DispatchError;
 
+#[test]
+fn verify_pallet_index() {
+	assert_eq!(System::index(), 0);
+	assert_eq!(moonbeam_runtime::Utility::index(), 1);
+	assert_eq!(moonbeam_runtime::Timestamp::index(), 2);
+	assert_eq!(Balances::index(), 3);
+	assert_eq!(moonbeam_runtime::Sudo::index(), 4);
+	assert_eq!(moonbeam_runtime::RandomnessCollectiveFlip::index(), 5);
+	assert_eq!(moonbeam_runtime::ParachainSystem::index(), 6);
+	assert_eq!(
+		<moonbeam_runtime::Runtime as frame_system::Config>::PalletInfo::index::<
+			moonbeam_runtime::TransactionPayment,
+		>(),
+		Some(7)
+	);
+	assert_eq!(moonbeam_runtime::ParachainInfo::index(), 8);
+	assert_eq!(moonbeam_runtime::EthereumChainId::index(), 9);
+	assert_eq!(moonbeam_runtime::EVM::index(), 10);
+	assert_eq!(
+		<moonbeam_runtime::Runtime as frame_system::Config>::PalletInfo::index::<
+			moonbeam_runtime::Ethereum,
+		>(),
+		Some(11)
+	);
+	assert_eq!(moonbeam_runtime::ParachainStaking::index(), 12);
+	assert_eq!(moonbeam_runtime::Scheduler::index(), 13);
+	assert_eq!(moonbeam_runtime::Democracy::index(), 14);
+	assert_eq!(
+		<moonbeam_runtime::Runtime as frame_system::Config>::PalletInfo::index::<
+			moonbeam_runtime::CouncilCollective,
+		>(),
+		Some(15)
+	);
+	assert_eq!(
+		<moonbeam_runtime::Runtime as frame_system::Config>::PalletInfo::index::<
+			moonbeam_runtime::TechComitteeCollective,
+		>(),
+		Some(16)
+	);
+	assert_eq!(
+		<moonbeam_runtime::Runtime as frame_system::Config>::PalletInfo::index::<
+			moonbeam_runtime::Treasury,
+		>(),
+		Some(17)
+	);
+	assert_eq!(moonbeam_runtime::AuthorInherent::index(), 18);
+	assert_eq!(moonbeam_runtime::AuthorFilter::index(), 19);
+	assert_eq!(CrowdloanRewards::index(), 20);
```

Comment:
Here you've imported the Pallet and called it directly but in most places you use the fully qualified `moonbase_runtime::` prefix.

I don't care much which way we do it, but they should probably be consistent.

Reply:
I only used the direct import when it was imported for other tests. Otherwise, I used fully qualified to just import it here because it isn't used anywhere else in the tests.